micronota/commands/annotate.py

```diff
@@ -6,17 +6,32 @@
 # The full license is in the file COPYING.txt, distributed with this software.
 # ----------------------------------------------------------------------------
 
+from os.path import join, splitext, basename
 import click
 
+from ..workflow import annotate
+
 
 @click.command()
-@click.option('-i', '--input_fp', type=click.File('r'),
+@click.option('-i', '--input_fp', type=str,
```

Comment:
because I'd like it to be a str instead of opened file object. but I find `click.Path` is better for the purpose.